CryptographyExercise/CryptographyExerciseTests/CryptographyExersiseTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using CryptographyExercise;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Security.Cryptography;
+
+namespace CryptographyExercise.Tests
+{
+    /// <summary>
+    /// This class cobntains methods for testing the methods in the <see cref="CryptographyExersise"/> class.
```

Comment:
The word "contains" is misspelled in the comment. Remember that XML comments will show up in intellisense and potentially in customer facing documentation, so it's good to ensure you spell things correctly.